src/Nest/Domain/Stats/NodeStats.cs

```diff
@@ -167,8 +167,11 @@ public class MemoryStats : BaseMemoryStats
 			public int UsedPercent { get; internal set; }
 			[JsonProperty("actual_free")]
 			public string ActualFree { get; internal set; }
+			[Obsolete]
 			[JsonProperty("actual_free_in_bytes")]

```

Comment:
This `JsonProperty` is a duplicate and needs to be removed, the obsoleted property should get and set the new property `ActualFreeInBytes`.